### ISSUE TITLE
VPL merged tilesets behaviour at higher zoom levels

### ISSUE THREAD

**Issue by sastcsgh** [versatiles-org/versatiles-rs#128]
Currently, in a tileset merged through `from_vectortiles_merged`, if a certain zoom level does not exist for a specific source, versatiles does not return it. It would be brilliant if there would be some way to configure it to behave like MapLibre does when a maximum zoom level for a source is specified, namely to re-use the highest existing zoom level for zoom levels above this.

**Comment by MichaelKreil** [comment]
If I understand your issue correctly:
- Consider a scenario with data source A covering zoom levels 1 to 3 and data source B covering zoom levels 2 to 4.
- The expected behavior for from_vectortiles_merged is to provide merged vector tiles for zoom levels 1 through 4.

As far as I can tell, this should already work as described. Could you share a specific scenario where this isn’t the case?

**Comment by sastcsgh** [comment]
Yes, it provides merged vector tiles for zoom levels 1 through 4, in the sense that it returns the tiles of data source B for zoom level 4. However, on zoom level 4, data source A is no longer included.  What would be great (but probably not at all straightforward to implement) if the merged tileset would provide an overzoomed version of zoom level 3 of data source A, similar to the way that MapLibre does.

**Comment by MichaelKreil** [comment]
I've fixed a lot of problems since version 0.14.x. Now I can't reproduce your issue.
Is it fixed in the latest version 0.15.1?
